packages/table/src/headers/columnHeader.tsx

```diff
@@ -168,50 +205,98 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
         });
         const cellLoading = cell.props.loading != null ? cell.props.loading : loading;
         const isColumnSelected = Regions.hasFullColumn(selectedRegions, columnIndex);
-        const cellProps: IColumnHeaderCellProps = { className, isColumnSelected, loading: cellLoading };
+        const isColumnTemporarilyReorderable = this.isColumnTemporarilyReorderable(isColumnSelected);
```

Comment:
Yeah wasn't sure about this, but I wanted to more strongly disambiguate `isReorderable` from `isColumnReorderable`. It's not necessarily clear at a glance what the difference is.